### ISSUE TITLE
Wrapping/Generators/Python/Tests/extras.py fails with additional type wrapping

### ISSUE THREAD

**Issue by Leengit** [InsightSoftwareConsortium/ITK#2293]
### Description

A variable `vectorimage` was removed from `extras.py`.  However, this variable is still used when the cmake flags indicate additional wrapping.

### Steps to Reproduce

Build with cmake flags:
```
-DITK_WRAP_complex_double:BOOL=ON -DITK_WRAP_complex_float:BOOL=ON 
-DITK_WRAP_covariant_vector_double:BOOL=ON -DITK_WRAP_covariant_vector_float:BOOL=ON 
-DITK_WRAP_double:BOOL=ON -DITK_WRAP_float:BOOL=ON -DITK_WRAP_rgba_unsigned_char:BOOL=ON 
-DITK_WRAP_rgba_unsigned_short:BOOL=ON -DITK_WRAP_rgb_unsigned_char:BOOL=ON 
-DITK_WRAP_rgb_unsigned_short:BOOL=ON -DITK_WRAP_signed_char:BOOL=ON 
-DITK_WRAP_signed_long_long:BOOL=ON -DITK_WRAP_signed_short:BOOL=ON -DITK_WRAP_unsigned_char:BOOL=ON 
-DITK_WRAP_unsigned_long_long:BOOL=ON -DITK_WRAP_unsigned_short:BOOL=ON 
-DITK_WRAP_vector_double:BOOL=ON -DITK_WRAP_vector_float:BOOL=ON
```
### Expected behavior

ctest returns no failures.

### Actual behavior

`ctest -V -R PythonExtrasTest` gives
```
2694: Traceback (most recent call last):
2694:   File ".../Wrapping/Generators/Python/Tests/extras.py", line 393, in <module>
2694:     cast = vectorimage.astype(t[0])
2694: NameError: name 'vectorimage' is not defined
2694: itkTestDriver: Process exited with return value: 1
```

### Reproducibility

Always

### Versions

5.2.0

### Environment

Ubuntu 20.04

**Comment by thewtex** [comment]
@Leengit does replacing `vectorimage` with `image` address the issue?  This was mistakenly omitted in https://github.com/InsightSoftwareConsortium/ITK/commit/e2f61927f617c830dd17cb2cb2c73c1662e9c65c

**Comment by Leengit** [comment]
No, it looks like directly replacing `vectorimage` with `image` will not address the issue.  The test is testing the ability to cast VectorImages, and it depends upon its input `vectorimage` being `VectorImage<float,2>` (`VIF2`).  In contrast, it looks like `image` is `Image<Vector<float,3>,2>` (`IVF32`).

**Comment by thewtex** [comment]
After the refactoring, `image` is an `itk.VectorImage`.

**Comment by Leengit** [comment]
Resolved by Pull Request #2317.
